core/src/main/java/org/infinispan/remoting/transport/jgroups/ClusterView.java

```diff
@@ -17,21 +21,22 @@ public class ClusterView {
    static final int FINAL_VIEW_ID = Integer.MAX_VALUE;
 
    private final int viewId;
-   private final List<Address> members;
-   private final Set<Address> membersSet;
-   private final Address coordinator;
+   private final Map<Address, ExtendedUUID> view;
```

Comment:
I think we should store `JGroupsAddress` here as that's what's always returned by `JGroupsAddressCache` and it means we can update `getAddressFromView` to take a `JGroupsAddress` as the parameter.